book/src/installation_windows.md

```diff
@@ -62,6 +62,9 @@ Download Gettext 0.21 from [mlocati.github.io](https://mlocati.github.io/article
 
 ### Pkg-config
 
+> **Warning**
+> **To avoid any potential problems, we strongly recommend that you make sure that nothing takes precedence over pkg-config-lite. https://github.com/gtk-rs/gtk4-rs/issues/1257**
```

Comment:
To be more specific, maybe the instructions could recommend users to check that `where pkg-config` (legacy cmd) or `(Get-Command pkg-config).Path` (PowerShell) resolves to the correct binary.

Optionally - although this may be a bit too verbose - even verify that `echo %PKG_CONFIG_PATH%` (legacy cmd) or `echo $Env:PKG_CONFIG_PATH` (PowerShell) displays the expected output.

Reply:
Powershell alone is fine

Reply:
@Hofer-Julian Any preference on how we could phrase this while keeping it short?

Reply:
The guide currently doesn't mention PowerShell though, only cmd:

> From the Windows start menu, search for `x64 Native Tools Command Prompt for VS 2019`.

To use PowerShell I launched the Windows Terminal and picked the 'Developer PowerShell VS 2022' profile, which judging from its invocation I assume uses the x64 architecture:

```
powershell.exe -NoExit -Command "&{Import-Module """C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\Tools\Microsoft.VisualStudio.DevShell.dll"""; Enter-VsDevShell a6486c7f -SkipAutomaticLocation -DevCmdArguments """-arch=x64 -host_arch=x64"""}"
```

Reply:
I agree, I personally use cmd so we should cover both cases and not make assumptions

Reply:
Added @pieterdd recommendation to be more specific. Thank!

Reply:
Ah yeah my bad, I forgot that we have to rely on cmd for this. Can you please remove powershell then?

Reply:
> Ah yeah my bad, I forgot that we have to rely on cmd for this. Can you please remove powershell then?

Done! I think powershell works just as well, but that's untested territory on my part, so maybe we can add it later when we're more confident.

Reply:
Fair enough. It worked for me in PowerShell, but agree to stick with the safer route for now.